test/foundry/deploy/10_holograph_tests.t.sol

```diff
@@ -2,698 +2,759 @@
+  }
+
+  /* -------------------------------------------------------------------------- */
+  /*                                 SET BRIDGE                                 */
+  /* -------------------------------------------------------------------------- */
+
+  /**
+   * @notice Test the ability of the admin to alter the bridge slot in the Holograph contract.
+   * @dev This test is designed to verify that the admin can alter the bridge slot in the contract.
+   * This test performs a prank operation on the current admin address of the Holograph contract,
+   * and then sets a new random address as the bridge in the contract.
+   * Refers to the hardhat test with the description 'should allow admin to alter _bridgeSlot'
+   */
+  function testAllowAdminAlter_bridgeSlot() public {
+    address random = RandomAddress.randomAddress();
+    vm.prank(origin);
+    holograph.setBridge(random);
+  }
+
+  /**
+   * @notice Test the revert behavior when the owner tries to alter the bridge slot in the Holograph contract.
+   * @dev This test is designed to verify that only the admin can alter the bridge slot in the contract.
+   * This test expects a revert with the message 'HOLOGRAPH: admin only function' when the owner attempts to
+   * alter the bridge slot in the Holograph contract.
+   * Refers to the hardhat test with the description 'should fail to allow owner to alter _bridgeSlot'
+   */
+  function testAllowOwnerToAlter_bridgeSlotRevert() public {
+    address random = RandomAddress.randomAddress();
+    vm.expectRevert(bytes(ErrorConstants.ONLY_ADMIN_ERROR_MSG));
+    holograph.setBridge(random);
+  }
+
+  /**
+   * @notice Test the revert behavior when a non-owner tries to alter the bridge slot in the Holograph contract.
+   * @dev This test is designed to verify that only the admin can alter the bridge slot in the contract.
+   * This test expects a revert with the message 'HOLOGRAPH: admin only function' when a non-owner attempts to
+   * alter the bridge slot in the Holograph contract.
+   * Refers to the hardhat test with the description 'should fail to allow non-owner to alter _bridgeSlot'
+   */
+  function testAllowNonOwnerToAlter_bridgeSlotRevert() public {
+    address random = RandomAddress.randomAddress();
+    vm.expectRevert(bytes(ErrorConstants.ONLY_ADMIN_ERROR_MSG));
+    vm.prank(user);
+    holograph.setBridge(random);
+  }
+
+  /* -------------------------------------------------------------------------- */
+  /*                                 GET CHAINID                                */
+  /* -------------------------------------------------------------------------- */
+
+  /**
+   * @notice Test the validity of the chainId slot in the Holograph contract.
+   * @dev This test is designed to ensure the correct functionality of the chainId slot in the contract.
+   * This test verifies that the value returned by `getChainId()` in the Holograph contract
```

Comment:
return that the value set in initialization run ok

Reply:
Done

Reply:
Done